content/en/blog/_posts/2022-04-19-csi-nodeexpandsecret.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Use Secrets for Node-Driven Resize of CSI Volumes."
+date: 2022-07-26T10:00:00-08:00
+slug: kubernetes-1-25-use-secrets-while-expanding-csi-volumes-on-node-alpha
+---
+
+**Author:** Humble Chirammal (Red Hat), Louis Koo (deeproute.ai)
+
+Kubernetes v1.25, released earlier this month, introduced a new feature
+that lets your cluster expand storage volumes, even when access to those
+volumes requires a secret (for example: a credential for accessing a SAN fabric)
+to perform node expand operation. This new behavior is in alpha and you
+must enable a feature gate (`CSINodeExpandSecret`) to make use of it.
+You must also be using [CSI](https://kubernetes-csi.github.io/docs/)
+storage; this change isn't relevant to storage drivers that are built in to Kubernetes.
+
+TO turn on this new, alpha feature, you enable the `CSINodeExpandSecret` feature
+gate for the kube api server and kubelet, provide a mechanism to receive secretRef
+configuration as part of NodeExpansion  by the CSI drivers thus make use of
+the same to perform node side expansion operation with the underlying
+storage system.
```

Comment:
I'm worried that readers who have only read up to this points won't know what the new feature does. I'll try to suggest some different wording here.